doc/reference/tooling/tt_cli/connect.rst

```diff
@@ -28,6 +28,36 @@ Options
 
     ``-`` – read the script from stdin.
 
+.. option:: -i, --interactive
+
+    Enter the interactive mode after evaluating the script passed in ``-f``/``--file``.
+
+..  option:: -l LANGUAGE, --language LANGUAGE
+
+    The input language of the :ref:`tt interactive console <tt-interactive-console>`:
+    ``lua`` (default) or ``sql``.
+
+..  option:: -x FORMAT, --outputformat FORMAT
+
+    The output format of the :ref:`tt interactive console <tt-interactive-console>`:
+    ``yaml`` (default), ``lua``, ``table``, ``ttable``.
+
+..  option:: --sslcertfile FILEPATH
+
+    The path to a trusted certificate authorities (CA) file for encrypted connections.
+
+..  option:: --sslcafile FILEPATH
+
+    The path to an SSL certificate file for encrypted connections.
```

Comment:
Looks like the descriptions for `--sslcafile` and `--sslcertfile` should be swapped.